packages/editor-ui/src/views/SettingsUsageAndPlan.vue

```diff
@@ -85,7 +201,52 @@ onMounted(() => {
 	}
 }
 
-div[class*="info"] span {
+.chart {
+	display: flex;
+  align-items: center;
+	justify-content: flex-end;
+	flex-grow: 1;
+
+	.line {
+		display: block;
+		height: 10px;
+		width: 100%;
+		max-width: 260px;
+		margin: 0 var(--spacing-m);
+		border-radius: 10px;
+		background: var(--color-background-base);
+
+		.bar {
+			float: left;
+			height: 100%;
+			max-width: 100%;
+			background: var(--color-secondary);
+			border-radius: 10px;
+			transition: width 0.2s $ease-out-expo;
+		}
+	}
+}
+
+div[class*="info"] > span > span:last-child {
 	line-height: 1.4;
+	padding: 0 0 0 var(--spacing-4xs);
+}
+</style>
+
+<style lang="scss" scoped>
+:deep(.el-dialog__wrapper) {
```

Comment:
we should avoid this as much as possible..

Reply:
this is just a comment.. we don't have to change anything here.. I get sometimes this cannot be avoided